Common/src/main/java/at/petrak/hexcasting/common/blocks/decoration/BlockHexFenceGate.java

```diff
@@ -0,0 +1,32 @@
+package at.petrak.hexcasting.common.blocks.decoration;
+
+import at.petrak.hexcasting.annotations.SoftImplement;
+import at.petrak.hexcasting.common.lib.HexBlockSetTypes;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.block.FenceGateBlock;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.properties.WoodType;
+
+public class BlockHexFenceGate extends FenceGateBlock {
+
+    public BlockHexFenceGate(Properties $$0) {
+        super($$0, WoodType.DARK_OAK);
+    }
```

Comment:
```java
public BlockHexFenceGate(Properties $$0) {
    super($$0, WoodType.DARK_OAK);
}
```
should probably give this a more descriptive name than $$0. I mean i guess it doesn't super matter since it's a very generic constructor but still.

Same thing on BlockHexFence.